### PR TITLE
fix preview, preview on title click

### PR DESCRIPTION
**Pull request by bumjung**

### CODE REVIEW

**Comment by insoochoo** [comment]
should we do click title? or click the whole li?

File: views/qube.html [hunk 1]
```diff
@@ -159,13 +159,13 @@
                               </div>
                               <div class="videoContainer">
                                   <div class="video" ng-repeat="video in ytSearchResult">
-                                    <div class="video-image tooltip-right noselect" id="{{video.id}}" ng-click="onPreviewClick(video.snippet.title)" onclick="onPreviewClick(this)" data-tooltip="Preview video">
```
Comment (insoochoo) [inline views/qube.html]: let's also move the tooltip

**Comment by bumjung** [comment]
@insoochoo I think li is too large of an area; we can keep it as just text and review inspectlet if they're clicking other areas

**Comment by insoochoo** [comment]
👍 💯 ☕️

**Pull request merged**